modules/search/class.jetpack-search-helpers.php

```diff
@@ -227,6 +227,10 @@ static function get_date_filter_type_name( $type, $is_updated = false ) {
 	static function generate_widget_filter_name( $widget_filter ) {
 		$name = '';
 
+		if ( ! is_array( $widget_filter ) || ! array_key_exists( 'type', $widget_filter ) ) {
```

Comment:
You could simplify this check if you want

```suggestion
		if ( ! isset( $widget_filter['type'] ) ) {
```